File: ClavaWeaver/resources/clava/test/weaver/c/src/scope.c
void numStatements() {
	// A comment, does not count
	int a = 0, b = 0;
	
	#pragma does not count
	for(int i=0; i<10; i++) {
		a++;
	}
	
	if(a) {
		b = 1;
	} else {
		b = 2;
	}
}

File: ClavaWeaver/resources/clava/test/weaver/Scope.js
import Query from "@specs-feup/lara/api/weaver/Query.js";

// Count statements
const $body = Query.search("function", "numStatements").first().body;
console.log("numStatements (depth): " + $body.getNumStatements());
console.log("numStatements (flat): " + $body.getNumStatements(true));
